sdk/contentsafety/ai-content-safety-rest/README.md

```diff
@@ -138,10 +161,10 @@ if (isUnexpected(result)) {
   throw result;
 }
 
-console.log("Hate severity: ", result.body.hateResult?.severity);
-console.log("SelfHarm severity: ", result.body.selfHarmResult?.severity);
-console.log("Sexual severity: ", result.body.sexualResult?.severity);
-console.log("Violence severity: ", result.body.violenceResult?.severity);
+for (let i = 0; i < result.body.categoriesAnalysis.length; i++) {
+  const textCategoriesAnalysisOutput: TextCategoriesAnalysisOutput = result.body.categoriesAnalysis[i];
```

Comment:
```suggestion
  const textCategoriesAnalysisOutput = result.body.categoriesAnalysis[i];
```
In RLC, we don't encourage customer to use type annotation, did you see any errors if we don't add the type annotation here ?

Reply:
Checked all samples and remove type annotation.